libs/pages/settings/src/lib/ui/page-organization-helm-repositories/crud-modal/crud-modal.tsx

```diff
@@ -0,0 +1,185 @@
+import { type AvailableHelmRepositoryResponse, type HelmRepositoryResponse } from 'qovery-typescript-axios'
+import { Controller, useFormContext } from 'react-hook-form'
+import { ExternalLink, InputSelect, InputText, InputTextArea, InputToggle, ModalCrud } from '@qovery/shared/ui'
+
+export interface CrudModalProps {
+  repository?: HelmRepositoryResponse
+  availableHelmRepositories: AvailableHelmRepositoryResponse[]
+  onSubmit: () => void
+  onClose: () => void
+  loading?: boolean
+  isEdit?: boolean
+}
+
+export function CrudModal({ isEdit, onSubmit, onClose, loading, availableHelmRepositories }: CrudModalProps) {
+  const { control, watch } = useFormContext()
+
+  return (
+    <ModalCrud
+      title={isEdit ? 'Edit helm repository' : 'Set helm repository'}
+      onSubmit={onSubmit}
+      onClose={onClose}
+      loading={loading}
+      isEdit={isEdit}
+      howItWorks={
+        <>
+          <p>
+            You can define here the repository (HTTPS or OCI) that you want to use within your organization to deploy
+            your helm charts.
+          </p>
+          <p>You can create a new repository by defining:</p>
+          <ul className="list-disc list-inside ml-3 mb-2">
+            <li>its name and description</li>
+            <li>kind (HTTPS or OCI)</li>
+            <li>the repository URL (Starting with https:// or oci://)</li>
+            <li>the credentials</li>
+            <li>the skip TLS verification option (to activate the helm argument --insecure-skip-tls-verify)</li>
+          </ul>
+          <ExternalLink
+            className="mt-2"
+            href="https://hub.qovery.com/docs/using-qovery/configuration/organization/helm-repository/"
```

Comment:
Alessandro can share the real link or not available?

Reply:
Not yet available. I'll update it when I have it.